config/initializers/mini_magick.rb

```diff
@@ -1,3 +1,3 @@
 MiniMagick.configure do |config|
-  config.graphicsmagick = true
+  config.graphicsmagick = Rails.env.production?
```

Comment:
Why? 
Well I know why but maybe installing graphicsmagick instead of changing the local vs the prod environment ist the better solution ? :wink: